packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -22,10 +22,18 @@ describe('transform traces to ddg paths', () => {
     })),
     attributes: [{ key: 'exemplar_trace_id', value: trace.data.traceID }],
   });
-  const makeSpan = (spanName, parent, kind) => ({
+
+  const addExemplarTraceIDs = (path, traces) => {
```

Comment:
Why do we use `const`+lambda instead of `function`? I find function a lot more readable & explicit than a lambda.

Reply:
I just followed the other parts of this code, I think there is no special reason for using lambda.